src/playbacks/html5_video/html5_video.js

```diff
@@ -140,12 +140,19 @@ export default class HTML5Video extends Playback {
     this._handleBufferingEvents()
     this.trigger(Events.PLAYBACK_LOADEDMETADATA, {duration: e.target.duration, data: e})
     this._updateSettings()
+    this._checkHighDefinition(e)
     const autoSeekFromUrl = typeof(this._options.autoSeekFromUrl) === 'undefined' || this._options.autoSeekFromUrl
     if (this.getPlaybackType() !== Playback.LIVE && autoSeekFromUrl) {
       this._checkInitialSeek()
     }
   }
 
+  _checkHighDefinition(e) {
+    const currentHeight = e.target.videoHeight
+    this.highDefinition = currentHeight >= 720
```

Comment:
Ah I didn't realise the HD event was already implemented in the hls playback. This makes it a bit more complex as this needs to not conflict with that one. I.e the hls playback condition also checks the bitrate whereas here it doesn't.

Maybe the new method `_checkHighDefinition()` could also be implemented (overridden) in the hls playback and that would be where the event is fired there too?